### PR TITLE
add en_GB translation

### PR DESCRIPTION
**Pull request by abdolg**
translated the german content by kern.muenchen.

### CODE REVIEW

File: src/assets/locales/en/translations.json [hunk 1]
```diff
@@ -1,198 +1,205 @@
...
-  "career.error.max-number": "Maximal number of documents is reached.",
-  "career.error.max-size": "Maximal filesize of 20 MB is reached.",
-  "career.error.uploaded": "The document \"{{name}}\" was already uploaded.",
-  "career.action.alternative": "<0>Or send your application to <1>career@satellytes.com</1></0>",
-  "career.action.upload": "<0>Drop the files here or <1>browse</1></0>",
+  "career.error.max-number": "Maximum number of documents has been reached.",
+  "career.error.max-size": "Maximum document size (20MB) was exceeded",
+  "career.error.uploaded": "The document “{{name}}” has already been uploaded.",
+  "career.action.alternative": "<0>Or send your application to: <1>career@satellytes.com</1></0>",
+  "career.action.upload": "<0>Drag the documents onto the field or <1>select</1> them</0>",
```
Comment (georgiee) [inline src/assets/locales/en/translations.json:90]: Format looks good.

File: src/assets/locales/en/translations.json [hunk 2]
```diff
@@ -1,198 +1,205 @@
...
-  "career.action.again-text": "Please try again. In case it does not work, please sent your application to <1>career@satellytes.com</1>",
-  "career.action.missing": "Please fill in the missing fields",
-  "career.seo.title-detail": "Career - {{name}} | Satellytes",
-  "career.seo.description-detail": "Apply now to become a {{name}}! Take a look at our other vacancies. We look forward to receiving your application.",
-  "career.seo.title": "Career | Satellytes",
-  "career.seo.description": "We are looking for passionate developers! Take a look at our open positions. We look forward to receiving your application.",
+  "career.action.again-text": "Please try again. If it doesn’t work, send your application directly to <1>career@satellytes.com</1>",
+  "career.action.missing": "Please fill in all required fields",
+  "career.seo.title-detail": "Careers - {{name}} | Satellytes",
+  "career.seo.description-detail": "Apply now as {{name}}! Check out our other vacancies. We look forward to receiving your application.",
```
Comment (georgiee) [inline src/assets/locales/en/translations.json:97]: Cool that they recognized the technical structure of the i18n format.

**Review by georgiee — APPROVED** [review]
Straight from the agency, so I trust the content is good I even checked the technical format with attributes. Let's merge that. Thanks.

**Review by georgiee — CHANGES_REQUESTED** [review]
hold found some technical quirks with the upload

**Review by feedm3 — APPROVED** [review]
That reads so good 👍

**Review by feedm3 — DISMISSED** [review]
Hold because of hold from Georgios.

**Review by georgiee — APPROVED** [review]
The career form has issues with some file names. not related, let's merge 🙏 I will dismiss the block from fabian (thanks for the support) and then let's enjoy the language ✨

**Pull request merged**